src/Database/Schema/Table.php

```diff
@@ -14,789 +14,9 @@
  */
 namespace Cake\Database\Schema;
 
-use Cake\Database\Exception;
-use Cake\Database\Type;
-use Cake\Datasource\ConnectionInterface;
-
 /**
- * Represents a single table in a database schema.
- *
- * Can either be populated using the reflection API's
- * or by incrementally building an instance using
- * methods.
- *
- * Once created Table instances can be added to
- * Schema\Collection objects. They can also be converted into SQL using the
- * createSql(), dropSql() and truncateSql() methods.
+ * @deprecated Provided for backward compatibility. Use TableSchema instead.
  */
-class Table
+class Table extends TableSchema
```

Comment:
@thinkingmedia  It is not a performance issue. @markstory says that your change breaks existing applications. The following code would throw a fatal error.
```php
namespace App\Model\Table;

use Cake\ORM\Table;
use Cake\Database\Schema\Table as Schema

class Articles extends Table
{
    protected function _initializeSchema(Schema $schema) { // Fatal error, as TableSchema is not a subclass of Schema
    
    }
}

```